src/pages/ModelsIndex/ModelsIndex.tsx

```diff
@@ -145,19 +145,21 @@ export default function Models() {
           <strong className="models__count">
             {modelCount} {pluralize(modelCount, "model")}
           </strong>
-          <ButtonGroup
-            activeButton={queryParams.groupedby}
-            buttons={["Status", "Cloud", "Owner"].map((group) => ({
-              children: group,
-              key: group.toLowerCase(),
-              to: urls.models.group({
-                groupedby: group.toLowerCase() as ModelsGroupedBy,
-              }),
-            }))}
-            buttonComponent={Link}
-            label="Group by:"
-            noWrap
-          />
+          <span className="models__header-controls">
+            Group by:{" "}
```

Comment:
This label was previously passed as a prop to ButtonGroup, but the Vanilla component does not support a label so I've added it here instead.